tck/src/main/x/tck/comparison/Compare.x

```diff
@@ -175,4 +169,29 @@ class Compare {
             return value1.key <=> value2.key;
         }
     }
+
+    @Test
+    void compareArraysFormal() {
+        //Int[] a1 = [1, 2];
+        //Int[] a2 = [2, 1];
```

Comment:
These methods have been moved from their original location, but their contents have also been commented out. Is this intentional, because it may reduce code coverage testing for existing backend?